app/src/main/java/com/mapbox/vision/examples/models/UiSign.kt

```diff
@@ -414,7 +414,14 @@ data class UiSign(val signType: SignType, val signNum: SignNumber) {
         WarningLowFlyingAircraft(ukResourceName = "warning_low_flying_aircraft_uk"),
         WarningOpeningOrSwingBridge(ukResourceName = "warning_opening_or_swing_bridge_uk"),
         WarningRailwayCrossingWithBarriers(ukResourceName = "warning_railway_crossing_with_barriers_uk"),
-        WarningTrafficQueues(ukResourceName = "warning_traffic_queues_uk")
+        WarningTrafficQueues(ukResourceName = "warning_traffic_queues_uk"),
+
+        RegulatoryKeepLeftPicture(usResourceName = "regulatory_keep_left_picture"),
```

Comment:
please sort new resources with old ones. For sample `RegulatoryKeepRight...` exist and new one should be along with them

Reply:
we don't sort them. The order is the same as ML and Core uses. So it's easier to compare all platforms. @RingerJK